include/torch-mlir/Dialect/Torch/IR/TorchTypes.h

```diff
@@ -53,6 +53,9 @@ class BaseTensorType : public Type {
   /// convenient API.
   Type getOptionalDtype() const;
 
+  /// Get the raw optional sparse tensor encoding.
+  Attribute getOptionalSparsity() const;
```

Comment:
It could be my personal preference, but I feel like `getOptionalSparseEncoding` sounds clearer to me than `getOptionalSparsity`. When I saw the function call later, I wasn't sure what it was getting.

Reply:
I am reusing the existing API so I think it is better to keep the original name.